src/orderbook.js

```diff
@@ -164,7 +201,15 @@ OrderBook.EVENTS = [
   'offer_changed', 'offer_funds_changed'
 ];
 
-OrderBook.DEFAULT_TRANSFER_RATE = 1000000000;
+OrderBook.DEFAULT_TRANSFER_RATE = Amount.from_json(1000000000);
+
+OrderBook.NOTIFY_THROTTLE_TIME = 4000;
```

Comment:
running your code (with debounce instead of throttle) with a node.js process, its using way less resources than before, I am simultaneously subscribed to 6 very active autobridged books (+ 2 XRP books) - its not mobile but I am not seeing anything close to 1 second to do the autobridge calc, (i did see .7 sec once, and fairly regularly .2-.3 sec.

Using debounce, autobridged calculation is never being called more than once per book per ledger, though it could under heavy volume.